templates/repo/diff/compare.tmpl

```diff
@@ -214,7 +214,7 @@
 					<button class="ui button green show-form">{{.locale.Tr "repo.pulls.new"}}</button>
 				</div>
 			{{else if .Repository.IsArchived}}
-				<div class="ui warning message">
+				<div class="ui warning message gt-tc">
```

Comment:
Erm… Please wait for #25763.

Reply:
Is the css rule is going to be changed? `gt-tc`

Reply:
Oh I see rules are changing

Reply:
I have updated the code according to `gt-text-center`